src/views/MyProfile/index.js

```diff
@@ -497,7 +497,7 @@ class MyProfile extends Component {
                     justify="flex-start"
                     spacing={2}
                   >
-                    <Grid item xs={12} md={profileCardSize} lg={profileCardSize}>
+                    <Grid item xs={12} md={profileCardSize} lg={8}>
```

Comment:
currently the profile card size is actually something that is defined so that it can change card sizes all over the ui in one variable change. I have one of them hard coded for the sake of testing, however, I do believe that the "profileCardSize" variable is actually set to 8, so good thing you mentioned this. I should go back and set it back to "profileCardSize" instead of 8.

Reply:
fixed it and changed it to profileCardSize.